scripts/build-doc

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env node
+
+/* eslint-env node */
+
+require('./util/work-from-root')();
+
+var docBuilder = require('./util/doc-builder')({
```

Comment:
It is, but it was originally set up for 0.12 support for the sake of contributors.

We're considering bumping that to the LTS, but this isn't the place to make that change.